### DISCUSSION TITLE
After installing WiinUPro my bluetooth is not recognized by Windows

### DISCUSSION THREAD

**Discussion by ThatSquirrel** [KeyPuncher/WiinUPro#152] [Help]
Hello, everyone, noob question here, i believe.
Please help me.

If I go to settings->bluetooth it says  bluetooth is turned off. It works inside WiinUPro with ds4, but not with the Wiimote. Also, I cant connect any other Bluetooth device to the PC.

**Comment by KeyPuncher** [comment]
Were you already using the DS4 controller with other software like SCPToolkit before installing WiinUPro?

If I had to guess, I'd say you may have installed the Bluetooth driver (which is checked by default) during the SCP Driver installation phase of the installer which may have messed with your Bluetooth.

**Comment by Victorsitou** [comment]
I have the same problem, any steps to fix it?
